pygmt/tests/test_basemap.py

```diff
@@ -59,69 +46,28 @@ def test_basemap_power_axis():
     """
     fig = Figure()
     fig.basemap(
-        R=[0, 100, 0, 5000], J="x1p0.5/-0.001", B=['x1p+l"Crustal age"', "y500+lDepth"]
+        region=[0, 100, 0, 5000],
+        projection="x1p0.5/-0.001",
+        frame=['x1p+l"Crustal age"', "y500+lDepth"],
     )
     return fig
 
 
-@check_figures_equal()
+@pytest.mark.mpl_image_compare
 def test_basemap_polar():
     """
     Create a polar basemap plot.
     """
-    fig_ref, fig_test = Figure(), Figure()
-    # Use single-character arguments for the reference image
-    fig_ref.basemap(R="0/360/0/1000", J="P6i", B="afg")
-    fig_test.basemap(region=[0, 360, 0, 1000], projection="P6i", frame="afg")
-
-    return fig_ref, fig_test
+    fig = Figure()
+    fig.basemap(region=[0, 360, 0, 1000], projection="P8c", frame="afg")
+    return fig
 
 
-@check_figures_equal()
+@pytest.mark.mpl_image_compare
 def test_basemap_winkel_tripel():
     """
     Create a Winkel Tripel basemap plot.
     """
-    fig_ref, fig_test = Figure(), Figure()
-    fig_ref.basemap(R="90/450/-90/90", J="R270/25c", B="afg")
-    fig_test.basemap(region=[90, 450, -90, 90], projection="R270/25c", frame="afg")
-    return fig_ref, fig_test
-
-
-@check_figures_equal()
-def test_basemap_rose():
-    """
-    Create a map with coast and use basemap to add a rose.
-    """
-    fig_ref, fig_test = Figure(), Figure()
-    fig_ref.coast(R="127.5/128.5/26/27", W="1/0.5p")
-    fig_ref.basemap(Td="jBR+w5c")
-    fig_test.coast(region=[127.5, 128.5, 26, 27], shorelines="1/0.5p")
-    fig_test.basemap(rose="jBR+w5c")
-    return fig_ref, fig_test
-
-
-@check_figures_equal()
-def test_basemap_compass():
-    """
-    Create a map with coast and use basemap to add a compass.
-    """
-    fig_ref, fig_test = Figure(), Figure()
-    fig_ref.coast(R="127.5/128.5/26/27", W="1/0.5p")
-    fig_ref.basemap(Tm="jBR+w5c+d11.5")
-    fig_test.coast(region=[127.5, 128.5, 26, 27], shorelines="1/0.5p")
-    fig_test.basemap(compass="jBR+w5c+d11.5")
-    return fig_ref, fig_test
-
-
-@check_figures_equal()
-def test_basemap_map_scale():
-    """
-    Create a map with coast and use basemap to add a map scale.
-    """
-    fig_ref, fig_test = Figure(), Figure()
-    fig_ref.coast(R="127.5/128.5/26/27", W="1/0.5p")
-    fig_ref.basemap(L="jMB+c1+w10k+l+f")
-    fig_test.coast(region=[127.5, 128.5, 26, 27], shorelines="1/0.5p")
-    fig_test.basemap(map_scale="jMB+c1+w10k+f+l")
-    return fig_ref, fig_test
```

Comment:
You sure about removing these 3 tests? I know they're just aliases, but rose/compass/map_scale seems like a key part of many maps that we should test.

Reply:
The "contributing guides" were written based on the existing tests at that time, so I don't think we should take it as a strict rule. 

Of course, I can add these tests if you think they're necessary.

Reply:
I would add the tests back, but maybe take away the `fig.coast` part and make it purely `fig.basemap` if possible.